content/influxdb/cloud-dedicated/reference/cli/influxctl/database/create.md

```diff
@@ -9,10 +9,11 @@ menu:
 weight: 301
 related:
   - /influxdb/cloud-dedicated/admin/custom-partitions/define-custom-partitions/
+  - /influxdb/cloud-dedicated/admin/custom-partitions/partition-templates/
 ---
 
 The `influxctl database create` command creates a new database with a specified
-retention period in an InfluxDB Cloud Dedicated cluster.
+retention period in an {{< product-name omit=" Clustered" >}} cluster.
```

Comment:
Should this be `a`? "a clustered cluster"? Not quite sure how that gets resolved. This is in a few places.

Reply:
The product-name shortcode itself would resolve to InfluxDB Clustered, but "an InfluxDB Clustered cluster" doesn't read well, so we omit " Clustered" from the product name and it resolves as "an InfluxDB cluster."

Reply:
aaah I read right past the "omit" part. Thanks!